File: stephanie/Problema0110/Problema0110.java
package stephanie.Problema0110;

import java.util.Scanner;

public class Problema0110 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("Digite um peso na Terra: ");
        double peso = Double.valueOf(input.nextLine());
        
        System.out.println("1 - Mercúrio");
        System.out.println("2 - Vênus");
        System.out.println("3 - Marte");
        System.out.println("4 - Júpiter");
        System.out.println("5 - Saturno");
        System.out.println("6 - Urano");
        System.out.println("7 - Netuno");
        System.out.println("Digite o número de um planeta: ");
        int n = Integer.valueOf(input.nextLine());

        switch (n){
            case 1: System.out.print("Peso em Mercúrio: "+peso*0.37);
            break;
            case 2: System.out.print("Peso em Vênus: "+peso*0.88);
            break;
            case 3: System.out.print("Peso em Marte: "+peso*0.38);
            break;
            case 4: System.out.print("Peso em Júpiter: "+peso*2.64);
            break;
            case 5: System.out.print("Peso em Saturno: "+peso*1.15);
            break;
            case 6: System.out.print("Peso em Urano: "+peso*1.17);
            break;
            case 7: System.out.print("Peso em Netuno: "+peso*1.18);
            break;
            default: System.out.print("Entrada Inválida!");
            break;
        }
    }
}

File: stephanie/Problema0110/Problema0110.py
peso = input("Digite um peso na Terra: ")

print("1 - Mercúrio")
print("2 - Vênus")
print("3 - Marte")
print("4 - Júpiter")
print("5 - Saturno")
print("6 - Urano")
print("7 - Netuno")
n = input("Digite o número de um planeta: ")

if int(n)<=0:
    print("Entrada Inválida!")
elif int(n)==1:
    print("Peso em Mercúrio: ",float(peso)*0.37)
elif int(n)==2:
    print("Peso em Vênus: ",float(peso)*0.88)
elif int(n)==3:
    print("Peso em Marte: ",float(peso)*0.38)
elif int(n)==4:
    print("Peso em Júpiter: ",float(peso)*2.64)
elif int(n)==5:
    print("Peso em Saturno: ",float(peso)*1.15)
elif int(n)==6:
    print("Peso em Urano: ",float(peso)*1.17)
elif int(n)==7:
    print("Peso em Netuno: ",float(peso)*1.18)
elif int(n)>7:
    print("Entrada Inválida!")
